src/jupynium/events_control.py

```diff
@@ -522,6 +522,13 @@ def process_request_event(nvim_info: NvimInfo, driver, event):
         event[3].send(ret_obj)
         return True, None
 
+    elif event[1] == "kernel_connect_info":
+        driver.switch_to.window(nvim_info.window_handles[bufnr])
+        code = "return Jupyter.notebook.kernel.id"
+        ret_obj = driver.execute_script(code)
```

Comment:
can you give it a better name? `kernel_id = driver.execute_script(“Jupyter.notebook.kernel.id“)` should be enough